strawberryfields/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.20.0"
+__version__ = "0.20.0-dev"
```

Comment:
Is this right?
```suggestion
__version__ = "0.21.0-dev"
```

Reply:
Thanks, I think I'm getting a bit tired. 🤦